app/templates/components/forms/orders/order-form.hbs

```diff
@@ -81,52 +81,103 @@
           {{/if}}
         {{/each}}
       {{/each}}
+
       {{#if isPaidOrder}}
         <h4 class="ui horizontal divider header">
           <i class="ticket icon"></i>
           {{t 'Billing Information'}}
         </h4>
-        <div class="field">
-          <label class="required" for="company">{{t 'Company'}}</label>
-          {{input type='text' id='company' value=data.company}}
-        </div>
-        <div class="field">
-          <label for="taxBusinessInfo">{{t 'Tax ID or Business ID'}}</label>
-          {{input type='text' id='taxBusinessInfo' value=data.taxBusinessInfo}}
-        </div>
-        <div class="field">
-          <label class="required" for="address">{{t 'Address'}}</label>
-          {{textarea rows='2' id='address' value=data.address}}
-        </div>
-        <div class="field">
-          <label class="required" for="city">{{t 'City'}}</label>
-          {{input type='text' id='city' value=data.city}}
-        </div>
-        <div class="field">
-          <label for="state">{{t 'State/Province'}}</label>
-          {{input type='text' id='state' value=data.state}}
-        </div>
-        <div class="field">
-          <label class="required" for="zipcode">{{t 'Zip Code'}}</label>
-          {{input type='text' id='zip_code' value=data.zipcode}}
-        </div>
-        <div class="field">
-          <label class="required">{{t 'Select country'}}</label>
-          {{#ui-dropdown class="search selection" selected=data.country
-                        onChange=(action (mut data.country)) as |execute mapper|}}
-            {{input type='hidden' id='country' value=data.country}}
-            <div class="default text">{{t 'Select country'}}</div>
-            <i class="dropdown icon"></i>
-            <div class="menu">
-              {{#each countries as |country|}}
-                <div data-value="{{map-value mapper country.name}}" class="item">
-                  <i class="{{lowercase country.code}} flag"></i>
-                  {{country.name}}
-                </div>
-              {{/each}}
-            </div>
-          {{/ui-dropdown}}
-        </div>
+        {{#if isBillingInfoMandatory}}
```

Comment:
This should instead be toggled by order amount?

Reply:
I have checked the `if PaidOrder` above.

Reply:
Updated comment. Check

Reply:
> This should not be shown in free orders in any case. In paid orders, it should be shown unconditionally if billing info is mandatory. And if it is not, then it should be togglable by is_billing_enabled checkbox in order

Its the same happening here, Please check the else part. In the else there is another checkbox to toggle `is_billing_info`

Reply:
https://stackoverflow.com/questions/4226284/how-to-convince-a-colleague-that-code-duplication-is-bad